### PR TITLE
Add audio moment retrieval recipes

### PR DESCRIPTION
**Pull request by h-munakata**
In this PR, I add audio moment retrieval dataset (clotho-moment, unav-100, tut-sound events 2017).
In addition to this, I modify evaluate.py to align train.py

### CODE REVIEW

File: training/evaluate.py [hunk 1]
```diff
@@ -252,13 +252,13 @@ def compute_mr_results(epoch_i, model, eval_loader, opt, criterion=None):
...
         )
-    elif opt.dset_name == 'charades':
+    elif opt.dset_name in ['charades', 'clotho-moment', 'unav100', 'tut2017']:
         post_processor = PostProcessorDETR(
             clip_length=opt.clip_length, min_ts_val=0, max_ts_val=150,
             min_w_l=2, max_w_l=60, move_window_method="left",
             process_func_names=("clip_ts", "round_multiple")
         )
-    elif opt.dset_name in ['tacos', 'activitynet', 'youtube_highlight', 'clotho-moment', 'unav100', 'tut2017']:
+    elif opt.dset_name in ['tacos', 'activitynet', 'youtube_highlight']:
```
Comment (awkrail) [inline training/evaluate.py:261]: Why do you add ['clotho-moment', 'unav100', 'tut2017'] to both of L255 and 261? I think that you need to write only one place.

File: training/evaluate.py [hunk 2]
```diff
@@ -427,6 +427,7 @@ def check_valid_combination(dataset, feature):
         option_manager = BaseOptions(args.model, args.dataset, args.feature)
         option_manager.parse()
         opt = option_manager.option
+        os.makedirs(opt.results_dir, exist_ok=True)
```
Comment (awkrail) [inline training/evaluate.py:429]: Is this really necessary?

File: training/evaluate.py (hunk 1 above)
Comment (h-munakata) [reply training/evaluate.py:261]: This commit moved ['clotho-moment', 'unav100', 'tut2017'] from L261 to L255 to fix my bug. 
Therefore, the final script does not include ['clotho-moment', ...] in two places, but in only one place.

File: training/evaluate.py (hunk 2 above)
Comment (h-munakata) [reply training/evaluate.py:429]: Yes. If we remove this line, we need to make a new directory manually when we evaluate the model using unav100 after we have trained models for clotho-moment.

Originally, train.py includes ```option_manager.clean_and_makedirs()```, but if we try to evaluate the models using test split of clotho-moment, this line deletes the folder including the checkpoints.

**Review by h-munakata — COMMENTED** [review]
I replied to the comments.

File: training/evaluate.py (hunk 1 above)
Comment (awkrail) [reply training/evaluate.py:261]: Got it.

File: training/evaluate.py (hunk 2 above)
Comment (awkrail) [reply training/evaluate.py:429]: OK.

**Pull request merged**